docs/source/migration.mdx

```diff
@@ -4,31 +4,30 @@ title: Migrating to Apollo Server 3
 
 import { ExpansionPanel } from 'gatsby-theme-apollo-docs';
 
-**Apollo Server 3 is generally available.** The focus of this major-version release is to provide a lighter, nimbler core library as a foundation for future features and improved extensibility.
+Apollo Server 3 is generally available. The focus of this major-version release is to provide a lighter, nimbler core library as a foundation for future features and improved extensibility.
 
-**Many Apollo Server 2 users don't need to make any code changes to upgrade to Apollo Server 3**, especially if you use the "batteries-included" `apollo-server` library (as opposed to a [middleware-specific library](./integrations/middleware/)).
+**Many Apollo Server 2 users don't need to make any code changes to upgrade to Apollo Server 3!** This is especially likely if you use the "batteries-included" `apollo-server` library (as opposed to a [middleware-specific library](./integrations/middleware/)).
 
-This document explains which features _do_ require code changes and how to make them.
+This document explains which features _do_ require code changes and how to make them. If you encounter issues while migrating, please [let us know in the forums](https://community.apollographql.com/).
```

Comment:
I think filing an issue would be fine for now. I'll be better at observing and responding on GitHub.